### ISSUE TITLE
Add support for Brazilian Portuguese (pt-BR)

### ISSUE THREAD

**Issue by basyl-durnan** [CareEvolution/MyDataHelpsUI#333]
# About

Adding support for Brazilian Portugues (pt-BR) for the CLUES project.

# MDH UI and View Utils - Locale Definition

This creates the translations for the MDH UI components and view builder.

## Code Changes

In the [**MyDataHelpsUI** repo](https://github.com/CareEvolution/MyDataHelpsUI):

1. Copy the [English language file](https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/helpers/strings-en.ts) to make one for the new language.
2. Send out for translations.
3. Once merged, deploy a new NPM package with the updated strings.
4. Once the new MDH UI version is published, update [MDH View Builder](https://github.com/CareEvolution/MyDataHelpsViewBuilder) to reference it.
5. Ditto for the [MDH View Library](https://github.com/CareEvolution/MyDataHelpsViewLibrary).

Note: The view builder main branch is deployed to QA automatically. Once tested, another PR is necessary to merge to the `prod` branch and deploy to prod. This can be done outside a release cycle.

## Testing

From the [Localization Test Plan](https://docs.google.com/document/d/1blcMJfHzE3PJ7iuL_jBA7nkoafRsjhZAT5JB7pPetww/edit?usp=sharing):

* TBD (need new test cases for this)

**Comment by basyl-durnan** [comment]
We already have pt supported and I believe the other two repos already reference it since it is available in prod. 

@lynnfaraday do you know how I could verify?

**Comment by basyl-durnan** [comment]
Awaiting https://github.com/CareEvolution/MyDataHelpsUI/pull/361

**Comment by lynnfaraday** [comment]
@basyl-durnan I don't think we need to wait on that one. Those localization gaps exist in all existing languages too and it wasn't a showstopper.

**Comment by basyl-durnan** [comment]
So is the only thing left to do is update View Builder and View Library with the latest MDH UI version?

**Comment by lynnfaraday** [comment]
@basyl-durnan No, I think we need to actually add PT-PT translations. THEN we can make a release and update the view builder/library.

**Comment by basyl-durnan** [comment]
I thought we were just going to go with "pt" for now and it really be "pt-BR"? Apologies if I missed some conversation. 

Would that mean we need to do everything we just did for pt-BR for pt-PT?

**Comment by basyl-durnan** [comment]
Per https://careevolution.slack.com/archives/C080WSVFCKT/p1732734604813449, are we good to move this forward?

**Comment by basyl-durnan** [comment]
`pt` was actually `pt-BR` to start. Otherwise, this is now solved by:

https://github.com/CareEvolution/MyDataHelpsViewBuilder/pull/147
https://github.com/CareEvolution/MyDataHelpsViewLibrary/pull/32
